src/components/MaterialUI/muiThemes.js

```diff
@@ -0,0 +1,467 @@
+	},
+	MuiTabs: {
+		...theme.MuiTabs,
+		root: {
+			...theme.root,
+			backgroundColor: theme.palette.grey.light,
+			paddingTop: theme.spacing(2),
+			position: "relative",
+			borderRadius: theme.spacing(1, 0, 0, 0),
+			"&:before": {
+				content: '""',
+				display: "flex",
+				position: "absolute",
+				borderBottom: `1px solid ${theme.palette.primary.main}`,
+				width: "100%",
+				bottom: 0,
+				zIndex: 10,
+			},
+			"& > .MuiTabs-scroller": {
+				marginLeft: theme.spacing(2),
+			},
+		},
+	},
+	MuiTab: {
+		...theme.MuiTab,
+		root: {
+			...theme.root,
+			backgroundColor: theme.palette.common.white,
+			color: theme.palette.grey.light,
+			border: `1px solid ${theme.palette.grey.borders}`,
+			borderRadius: theme.spacing(0.5, 0.5, 0, 0),
+			zIndex: 10,
+			textTransform: "uppercase",
+			height: theme.spacing(4.8),
+			minHeight: theme.spacing(2.4),
+			"& .MuiIconButton-root": {
+				padding: 0,
+			},
+			"&.Mui-selected": {
+				color: theme.palette.primary.main,
+				borderColor: theme.palette.primary.main,
+				zIndex: 10000,
+				borderBottom: `1px solid ${theme.palette.common.white}`,
+				marginBottom: -1,
+			},
+			"&.MuiTab-labelIcon svg": {
+				marginBottom: 0,
+				marginRight: 10,
+			},
+			"& .MuiTab-wrapper": {
+				flexDirection: "row",
+			},
+		},
+		labelIcon: {
+			...theme.labelIcon,
+			height: theme.spacing(4.8),
+			minHeight: theme.spacing(2.4),
+			"& path:not([fill='none'])": {
+				fill: "currentColor",
+			},
+		},
+	},
+	MuiTable: {},
+	MuiTableHead: {
+		...theme.MuiTableHead,
+		root: {
+			...theme.root,
+			backgroundColor: theme.palette.grey.lighter,
+		},
+	},
+	MuiTableBody: {},
+	MuiTableCell: {
+		...theme.MuiTableCell,
+		root: {
+			...theme.root,
+			fontFamily: theme.typography.fontFamily,
+			fontSize: theme.typography.fontSize,
+			borderBottomColor: theme.palette.secondary.light,
+			"&$head": {
+				fontWeight: theme.typography.fontWeightSemiBold,
+			},
+		},
+	},
+	MuiTableRow: {},
+	MuiTableContainer: {},
+	MuiTablePagination: {},
+	MuiTableSortLabel: {},
+	MuiTableFooter: {},
+	MuiInputBase: {
+		input: {
+			borderRadius: 4,
+			position: "relative",
+			backgroundColor: theme.palette.background.paper,
+			border: "1px solid #ced4da",
+			fontSize: 12,
+			maxWidth: "200px",
+			padding: "6px 6px 6px 6px",
+			transition: theme.transitions.create(["border-color", "box-shadow"]),
+		},
+	},
+	MuiSelect: {
+		select: {
+			minWidth: "150px",
+			borderRadius: 4,
+			"&:focus": {
+				borderRadius: 4,
+				borderColor: "#80bdff",
+			},
+		},
+	},
+});
+
+const createThemes = (applicationTheme, themeDefinition) => {
+	const applicationMuiTheme = createMuiTheme(applicationTheme);
+	const muiTheme = createMuiTheme({
+		...themeDefinition,
+		...commonTheme,
+	});
+
+	muiTheme.props = setThemeProps(muiTheme);
+	muiTheme.overrides = setThemeOverrides(muiTheme);
+
+	return {
+		applicationMuiTheme: applicationMuiTheme,
+		muiTheme: muiTheme,
+	};
+};
+
+export default createThemes;
```

Comment:
What for you've added mui theme inside orc-shared? Themes should be defined in applications (there is a high chance that in different application theme elements will be different).

Reply:
See my comments, I think there will be common themes between applications. If something needs to be different, then, you will be able to do it customize in the application. I see no points to have same overrides in each our projetcts.

Reply:
If you want me to rename that file, I can do it, but I will not copy/paste the OMS file into Analytics as such as it will ask double maintenance everytime we need to add overrides or change typography for all of our applications. In the future, we will have four applications using orc-shared, so every time we will need to change the overrides, we will need to do it in four projects. I did it only to ease maintenance in the future.

Reply:
Maybe I can named it "commonThemes" or "basicThemes", not sure

Reply:
but, how you are expecting to use it? the point is that currently theme is passed from oms to orc-shared. and because of that MUI theme could be accessed in makeStyle hook in both solutions. However if we won't pass theme from app to orc-shared (let's say app didn't define it's own theme) it means that we won't be able to use theme from that App. or what it will be necessary to do is to directly define MUI theme for that app as Theme imported from orc-shared.

Anyway I'm strictly not agree that we have to define "default theme". the default theme is already defined by MUI just to enable possibility to use "fallback values". But, different application will have lot of differences in colors fonts in which global classes colors/fonts/sizes should be used etc. That's what I got from discussion with Alain and that's why we are defining theme inside application and not inside orc-shared